### ISSUE TITLE
Handle Dataview `property::` breadcrumbs in tasks

### ISSUE THREAD

**Issue by eohannes** [SkepticMystic/breadcrumbs#534]
**Is your feature request related to a problem? Please describe.**
I've noticed that Breadcrumbs does not seem to be aware of Dataview property breadcrumbs that are listed inside tasks. For example:
```
Note B
--- 
- [ ] this is a task. down:: [[Note C]]
- [x] this is another task. up:: [[Note A]] 
```
In this scenario, I'd like the hierarchy to run `Note A -> Note B -> Note C`. Currently, it does not. I think this is because Dataview attributes properties in tasks to the tasks themselves (which can be queried separately from notes) and not to their parent note. (Conversely, Dataview attributes properties of a note to all its task children.)

**Describe the solution you'd like**
I'd like Breadcrumbs to provide an option to attribute a note's tasks' breadcrumbs (i.e., only those in a registered hierarchy) to the note itself when building its graph.

**Describe alternatives you've considered**
Currently I use a number of nasty DataviewJS blocks to explore my graph and generate tables of tasks derived from breadcrumbs and additional Dataview properties. This works fine for most of my purposes but does not inform the Grid View at the top of each page.

(Side note: I'd love it if I could query implied properties in Dataview too; this would make the DataviewJS code much less nasty. Let me know if this is also worthy of a feature request.)

**Additional context**
I'm using the latest beta of Breadcrumbs, `4.0.107-beta`.

**Comment by SkepticMystic** [comment]
Thanks for the detailed issue! This seems reasonable, will work on sometime

**Comment by SkepticMystic** [comment]
This sounds like a stupid question, but is _dataview_ even aware of fields in that format? I thought they used a different syntax for inline-fields.
Breadcrumbs uses whatever dataview gives it, so if those fields aren't used by BC, I suspect DV isn't supplying them

**Comment by eohannes** [comment]
Yes, but they don't show up in the parent page's properties; they're individually attributed to the task objects you'd get from `dv.current().file.tasks`. I don't know anything about Breadcrumbs' internal architecture but I imagine you'd need to add a few lines in the property discovering code that trawl each page's tasks for properties and attribute them to the page itself.
